website/templates/website/project_detail.html

```diff
@@ -4,20 +4,47 @@
 {% block body %}
     <div class="container">
         {% include 'website/messages.html' %}
-        <a href="{% url 'website:project_list' %}">Back to all project selections</a><br><br>
+        <a href="{% url 'website:project_list' %}">Back to all project selections</a><br/><br/>
+        {% if project.status != 'PROCESSED' %}
```

Comment:
I thought at some point I passed an is_done variable which checks if the transformer is done as well as the project selector...